news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title:  "PrestaShop 1.7.6.0 is available!"
+subtitle: "You waited for it, here it is!"
+date:   2019-07-10 18:00:00
```

Comment:
Do you plan to publish it today ?

Reply:
Well, the plan was to make it available tonight on autoupgrade and the dotcom, but to wait for tomorrow to send the emails to the 2M PS newsletter subscriber + publish the blogposts in 8 languages on the dotcom.

If we make it available tonight we have to publish the release note so it's not a bit official. I'll check with Coline to be sure.

Reply:
Be aware that if you publish an article in the future it won't show up until you rebuild jekyll. I changed it to 17:00 to be sure